tools/apiview/emitters/cadl-apiview/.gitignore

```diff
@@ -0,0 +1,2 @@
+temp/
+cadl-output/
```

Comment:
Similar here we should try to only have the one gitignore file in the root.

Reply:
Agree. Merged these into the root .gitignore.